Formula/wsk.rb

```diff
@@ -0,0 +1,34 @@
+class Wsk < Formula
+  desc "OpenWhisk Command-Line Interface (CLI)"
+  homepage "http://openwhisk.org"
+  url "https://github.com/apache/incubator-openwhisk-cli/archive/0.9.0-incubating.tar.gz"
```

Comment:
Why not just download the binary, instead of trying to compile/build on the person's computer?
You can grab the binary from this url https://github.com/apache/incubator-openwhisk-cli/releases/download/0.9.0-incubating/OpenWhisk_CLI-0.9.0-incubating-mac-amd64.zip

Then you avoid any dependencies on `gradl`, `go`, or `go-bindata`

Also faster install no dependencies to download and then compile.

Reply:
Homebrew builds things from source and provides our own binaries to users, who are consequently very unlikely to ever see any of the build dependencies. Allows us control over the build process without compromising speed or dependency weight for users.